lib/trento_web/router.ex

```diff
@@ -86,6 +86,9 @@ defmodule TrentoWeb.Router do
     scope "/v1", TrentoWeb.V1 do
       pipe_through [:api_v1]
 
+      get "/profile", ProfileController, :show
```

Comment:
I would personally put these entries close to `/users`.
Just to have some logical order

Reply:
done